include/gatekeeper_lls.h

```diff
@@ -30,12 +30,6 @@
 #include "gatekeeper_net.h"
 #include "gatekeeper_ratelimit.h"
 
-extern int lls_logtype;
```

Comment:
We are dropping the `extern` definition of `lls_logtype`, but not the actual definition of `lls_logtype`.